src/hooks/use-focus-selector.js

```diff
@@ -0,0 +1,84 @@
+import {useContext, useEffect, useState} from 'react';
+import readline from 'readline';
```

Comment:
IIRC you remove readline on a previous version.

I may remove this using `stdin.on('data')` directly